### ISSUE TITLE
Gunship flyer

### ISSUE THREAD

**Issue by memepeoples** [Anuken/Mindustry-Suggestions#2505]
**Describe the content or mechanics you are proposing.**
Gunship type airunit

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**

**Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** *Failure to do so may result in your request being closed automatically.*

A gunship type air unit that circles around there target till it is dead or they have been targeting it for too long

1. - [x] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
2. - [x] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
3. - [x] I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by OnyxFinger** [comment]
is this a troll ? Please provide more description.

**Comment by oinkymoo** [comment]
there are already plenty of gun on the existing air units

**Comment by IIINSTINCT** [comment]
This literally the worst suggestion i have seen

**Comment by MrDuck557** [comment]
1. [X] I have not filled in the template

**Comment by memepeoples** [comment]
From some reason It only saved the title (sorry) my idea was a airunit that circles it target at a distance will laser / bullets that can shoot through multiple targets

**Comment by memepeoples** [comment]
The idea of it was to have a anti unit role of circle moving targets dealing high damage and being hard to hit by ground and sea

**Comment by OnyxFinger** [comment]
> The idea of it was to have a anti unit role of circle moving targets dealing high damage and being hard to hit by ground and sea

ground is already kinda underpowered, and you would need high speed to do that to naval units :flushed:

**Comment by GitHubUser-B** [comment]
> 1. [x]  I have not filled in the template

- [X]  I'm rude to others for no reason

**Comment by GitHubUser-B** [comment]
> This is literally the worst suggestion i have seen

The answer I gave the duck goes for you too.

**Comment by OnyxFinger** [comment]
@GitHubUser-B they're harsh, but this is kinda deserved..

He didn't even fill the template, THEN once we commented gave an explanation

**Comment by GitHubUser-B** [comment]
> @GitHubUser-B they're harsh, but this is kinda deserved..
> 
> He didn't even fill the template, THEN once we commented gave an explanation

Maybe, but they're not just with this guy, they're with everyone.

**Comment by genNAowl** [comment]
> The idea of it was to have a anti unit role of circle moving targets dealing high damage and being hard to hit by ground and sea

The current air trees already fulfills their purposes, and do not have space for any more units.  Frankly, I do not think a new unit tree should be added just to fulfill this niche. 

> need high speed

As OnyxFinger already mentioned, yes, a unit which dodges ground/naval units would either need high speed or long range. Both are disastrous to balance (Kiting, Rushing/Dodging Defenses in PVP), and are not really viable options. 

> hard to hit by ground and sea

Basically, you want an air unit to counter ground ones. However, the whole idea of air units are supposed to be units that trade DPS, health, and other such stats for the ability to fly over any obstruction. This means that air units should always lose to ground in a straight up confrontation (head to head, where DPS and health matter most). The idea of an air unit that is even more powerful than a ground one violates this rule, and makes ground units (specifically the dagger tree) even worse than they are already.

**Comment by oinkymoo** [comment]
> > @GitHubUser-B they're harsh, but this is kinda deserved..
> > He didn't even fill the template, THEN once we commented gave an explanation
> 
> Maybe, but they're not just with this guy, they're with everyone.

the templates exist for a reason. they make information far more organized. when someone still completely fails to provide a sufficient amount of well-organized information, people reasonably become upset. this is also why anuke will ignore any bug report that doesn't properly fill out the template in the main github repo.

**Comment by konstantys** [comment]
> > @GitHubUser-B they're harsh, but this is kinda deserved..
> > He didn't even fill the template, THEN once we commented gave an explanation
> 
> Maybe, but they're not just with this guy, they're with everyone.

Anuken clearly stated; fill out the template. Yes, the author admitted that it was kind of a typo; yet it isn't for "no reason" - its just in order to make life easier for anuken... This is just one man eho is already very busy... Yes, being toxic and not nice to others is bad; but just try to understand why the ppl are like this...

**Comment by OnyxFinger** [comment]
> > @GitHubUser-B they're harsh, but this is kinda deserved..
> > He didn't even fill the template, THEN once we commented gave an explanation
> 
> Maybe, but they're not just with this guy, they're with everyone.

almost every suggestion is stupid af, and many a tired of it.

**Comment by GitHubUser-B** [comment]
> > > @GitHubUser-B they're harsh, but this is kinda deserved..
> > > He didn't even fill the template, THEN once we commented gave an explanation
> > 
> > 
> > Maybe, but they're not just with this guy, they're with everyone.
> 
> almost every suggestion is stupid af, and many a tired of it.

I know, you feel smart around "idiots". (some really are)

**Comment by numer99** [comment]
why?????
air units that are already in the game?
???
